gridstatus/tests/base_test_iso.py

```diff
@@ -297,6 +302,9 @@ def _check_time_columns(self, df, instant_or_interval="interval"):
                 "instant_or_interval must be 'interval' or 'instant'",
             )
 
+        if skip_column_named_time:
```

Comment:
if `instant_or_interval` is `"interval"`, we can automatically skip the column named time. no need to add this param

Reply:
So we can remove "Time" from `time_cols`? I was considering that but was hesitant because this affects a large number of tests.

Reply:
yes, but only in the case where it is `interval`